storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -0,0 +1,32 @@
+[id="persistent-storage-vsphere"]
+= vSphere CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-vsphere
+
+toc::[]
+
+== Overview
+
+{product-title} can provision persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
+
+:FeatureName: vSphere CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage assets, {product-title} installs the vSphere CSI Driver Operator and the vSphere CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* *vSphere CSI Driver Operator*: By default, the Operator provides a storage class that you can use to create persistent volumes claims (PVCs). You also have the option to create the vSphere storage class as described in xref:../../storage/persistent_storage/persistent-storage-vsphere.adoc#persistent-storage-using-vsphere[Persistent storage using vSphere].
+
+* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs.
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision vSphere storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.
```

Comment:
> using an in-tree, or non-CSI, driver to provision vSphere storage
Here I think is non-CSI driver?

> in-tree driver
I think in-tree plug-in should better, @bobfuru Could you double check what we used in other place?

Reply:
Agree with @wei-duan, "in-tree plug-in" is better. @lpettyjo, we use this `[IMPORTANT]` snippet in other CSI docs, so feel free to update those as well. (I realize that's not in scope for this PR, however.)